css/style.css

```diff
@@ -1,8 +1,22 @@
 .container {
+    background-color: #eddcd2;
 
 }
+.main {
+    padding: 3px;
+    background-color: #fff1e6;
+}
 .checkBox {
     margin: 0px 8px 0px 8px;
     text-align: center;
 
+}
+.title {
+    background-color: #b7b7a4;
+    text-align: center;
+    border-radius: 5px;
+    margin: 3px;
+    margin-top: 5px;
+    color: #362c28;
+    padding: 5px
 }
```

Comment:
overall style comments:
I don't really like the white bars on the side of the page. I think they should match the background color in the middle, but they could be a different color if desired. Regardless, they should not be left un-styled as they currently are.

optional formatting suggestions (feel free to ignore):

- add new lines between css classes for better readability.
- add new line at end of file.

Reply:
I agree with both of the optional comments.  

As well I agree with Thads feedback on the white space surrounding the app.  We could probably add in some images around these areas, or choose to extend the app to go to the whole page.  I think adding in images in this space would be the better option here

Reply:
white space will be addressed when desktop layout is completed, current layout is based on using iPhone X in dev tools.